swift-sdk/IterableAPI.swift

```diff
@@ -112,7 +112,7 @@ import UIKit
      - onFailure:   OnFailureHandler to invoke if token registration fails
      */
     @objc(registerToken:onSuccess:OnFailure:)
-    public static func register(token: Data, onSuccess: OnSuccessHandler? = nil, onFailure: OnFailureHandler? = nil) {
```

Comment:
We should keep = nil here because then Swift code can call without specifying both success and failure handlers.